test/service/test_request_validation.py

```diff
@@ -177,3 +181,60 @@ def test_manifest(self):
+
+    @mock_sts
+    @mock_s3
+    def test_bdbag_manifest(self):
+        """
+        As in test_manifest, moto will mock the requests.get call so we can't hit localhost; add_passthru let's us hit
+        the server (see GitHub issue and comment: https://github.com/spulec/moto/issues/1026#issuecomment-380054270)
+        """
+        logging.getLogger('test_request_validation').warning('test_manifest is invoked')
+        with ResponsesHelper() as helper, TemporaryDirectory() as zip_dir:
+            helper.add_passthru(self.base_url)
+            storage_service = StorageService()
+            storage_service.create_bucket()
+            url = self.base_url + '/repository/files/export?filters={"file":{}}&format=bdbag'
+            response = requests.get(url)
+            self.assertEqual(200, response.status_code, 'Unable to download manifest')
+            with ZipFile(response.text, 'r') as zip_fh:
+                zip_fh.extractall(zip_dir)
+            zip_fname = os.path.basename(os.path.splitext(response.text)[0])
+            with open(os.path.join(zip_dir, zip_fname, 'data', 'participant.tsv'), 'r') as fh:
+                participant = csv.reader(fh, delimiter='\t')
+                expected = ['entity:participant_id', '7b07b9d0-cc0e-4098-9f64-f4a569f7d746']
+                observed = []
+                for row in participant:
+                    observed.append(row[0])
+                self.assertEqual(len(observed), 2, 'participant.tsv contains duplicates.')
+                self.assertListEqual(sorted(expected), sorted(observed), 'participant.tsv contains incorrect data')
+
+            with open(os.path.join(zip_dir, zip_fname, 'data', 'sample.tsv'), 'r') as fh:
+
+                expectations = [('entity:sample_id','DID_scRSq06'),
+                                ('participant_id', '7b07b9d0-cc0e-4098-9f64-f4a569f7d746'),
+                                ('cell_suspension_id', '412898c5-5b9b-4907-b07c-e9b89666e204'),
+                                ('bundle_uuid', 'aaa96233-bf27-44c7-82df-b4dc15ad4d9d'),
+                                ('bundle_version', '2018-11-02T113344.698028Z'),
+                                ('file_content_type', 'application/gzip; dcp-type=data'),
+                                ('file_name', 'SRR3562915_1.fastq.gz'),
+                                ('file_sha256', '77337cb51b2e584b5ae1b99db6c163b988cbc5b894dda2f5d22424978c3bfc7a'),
+                                ('file_size', '195142097'),
+                                ('file_uuid', '7b07f99e-4a8a-4ad0-bd4f-db0d7a00c7bb'),
+                                ('file_version', '2018-11-02T113344.698028Z'),
+                                ('file_indexed', 'False'),
+                                ('file_url', config.dss_endpoint + '/files/7b07f99e-4a8a-4ad0-bd4f-db0d7a00c7bb?'
+                                                                   'version=2018-11-02T113344.698028Z&replica=gcp')
+                ]
```

Comment:
For aligned indent the closing delimiter is at the end of the last line. For hanging indent it's on a line of it's own. You are mixing both styles here.